ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -216,18 +217,12 @@ Create Openvino Models
         Set Suite Variable    ${MODEL_CREATED}    ${TRUE}
     END
 
+
 Model Serving Suite Teardown
     [Documentation]    Suite teardown steps after testing DSG. It Deletes
     ...                all the DS projects created by the tests and run RHOSi teardown
-    # Even if kw fails, deleting the whole project will also delete the model
-    # Failure will be shown in the logs of the run nonetheless
-    IF    ${MODEL_CREATED}
-        Clean All Models Of Current User
```

Comment:
@FedeAlonso Thanks for the PR. 
Can you run all sanity tests of 420 suite, and verify that the `Model Serving Suite Teardown` works for all  ?
Specifically, replacing the KW `Clean All Models Of Current User` in that teardown, with `Delete All DS Projects With Name Like    ${PRJ_TITLE}` might not work as expected (the current user might have other projects to be deleted, or similar project names of another user should not be deleted).

Reply:
@manosnoam I've noticed that we were using the DSProject name "model-serving-project" in multiple suites, so what I've done here is using the DSProject name "model-serving-project-420" as a base name (depending on the test it uses this one or adds something at the end), because it's a good practice to isolate the resources for each suite or test (preventing race conditions and this kind of hazards). So now almost each of this tests creates it's own DSProject.
With that modification, I've found that when trying to execute all this tests in a small cluster, It ran out of resources. So what I've done is that the deletion of the DSProjects comes as a Test Teardown instead of a Suite Tear Down.
I've also checked that the models are deleted when the DSProject is deleted and it works too.
I'll post the Test results below.